package.json

```diff
@@ -5,6 +5,7 @@
   "version": "6.61.0",
   "main": "dist/main.node.js",
   "browser": "dist/main.js",
+  "type": "module",
```

Comment:
just a heads up that we won't be able to load this in our React app because it's also not ESM.